libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
as all '_test.go' files are normally compiled when doing 'go test' I like to put common test helpers into something like `common_test.go` or `helpers_test.go` or `support_test.go` . Maybe have `api_common/helper/support_test.go` if helpers are 'local'. Some common scheme would make sense. Don't like big _test.go with loads of helper code I have search for the tests in first place.